ui/mirage/handlers/oidc-config.js

```diff
@@ -1,31 +1,31 @@
 export default function (server) {
   // built-in default key
-  server.get('/identity/oidc/key/default', () => {
-    return {
-      request_id: '5eeb2a46-8726-7336-76ee-8daea2860624',
-      lease_id: '',
-      renewable: false,
-      lease_duration: 0,
-      data: {
-        algorithm: 'RS256',
-        allowed_client_ids: ['*'],
-        rotation_period: 86400,
-        verification_ttl: 86400,
-      },
-      wrap_info: null,
-      warnings: null,
-      auth: null,
-    };
-  });
+  // server.get('/identity/oidc/key/default', () => {
```

Comment:
What's all this about?

Reply:
Originally the `default` key wasn't returning from the backend so I mocked it with mirage. When revisiting, I wanted to see if that was no longer the case (we've since made some changes to the handler, etc) and wanted to make sure all the tests passed both locally and circle CI - seems like it's safe to remove officially! 🎉